templates/software-partnerships/index.html

```diff
@@ -0,0 +1,389 @@
+{% extends "software-partnerships/base_software-partnerships.html" %}
+
+{% block title %}Software partnerships{% endblock %}
+
+{% block meta_description %}
+
+{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1ucKfDzLlw49sEM9OfkZ9aT5EHoDt4DsNebQ4ohzKn_E/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip is-deep">
+  <div class="row--50-50">
+    <div class="col">
+      <h1 class="p-heading--bold p-heading--2">
+        Turn your software into enterprise-grade solutions
+      </h1>
+    </div>
+    <div class="col">
+      <p>Give your customers peace of mind. Deliver applications on any cloud, with built-in resilience, automation and security.
+      </p>
+      <p>Explore our partnership opportunities for software vendors.</p>
+      <p>
+        <a href="/contact-us/form?product=operator-collection" class="p-button--positive">Contact us</a>
```

Comment:
Doesn't look like we have content for `operator-collection`. Should this be a new form?

Reply:
I have asked in the doc